### PR TITLE
[added] Translated global strings in french #4169

### PR DESCRIPTION
**Pull request by akk312000**
Added global strings translated to French language , Issue: #4169

### CODE REVIEW

File: frontend/assets/translations/fr.json [hunk 1]
```diff
@@ -1,921 +1,918 @@
...
-                "copy":"Copy",
-                "saving":"Saving",
-                "noResultsFor":"No results for",
-                "noteTaken":"Thank you, we've taken a note of that!",
-                "goToAllDatasources":"Go to all Datasource",
-                "send":"Send"
+  "globals": {
+    "readDocumentation": "Lire la documentation",
+    "cancel": "Annuler",
+    "save": "sauvegarder",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:5]:
```suggestion
    "save": "Sauvegarder",
```

File: frontend/assets/translations/fr.json [hunk 2]
```diff
@@ -1,921 +1,918 @@
...
+      "un": "ONU",
+      "youDontHaveany": "Tu n'as pas",
+      "youAreCaughtUp": "Vous êtes tous rattrapés!",
+      "view": "Voir"
+    },
+    "organization": {
+      "loadOrganizations": "Charges de chargement",
+      "createWorkspace": "Créer un espace de travail",
+      "workspaceName": "Nom de l'espace de travail",
+      "editWorkspace": "EDIT",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:211]:
```suggestion
      "editWorkspace": "Éditer l'espace de travail",
```

File: frontend/assets/translations/fr.json [hunk 3]
```diff
@@ -1,921 +1,918 @@
...
+      "menus": {
+        "addWorkspace": "Ajouter un espace de travail",
+        "menusList": {
+          "manageUsers": "gérer les utilisateurs",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:215]:
```suggestion
          "manageUsers": "Gérer les utilisateurs",
```

File: frontend/assets/translations/fr.json [hunk 4]
```diff
@@ -1,921 +1,918 @@
...
+          "manageEnv": "Gérer les variables d'environnement"
+        },
+        "manageUsers": {
+          "usersAndPermission": "Utilisateurs et autorisations",
+          "inviteNewUser": "Inviter un nouvel utilisateur",
+          "name": "NOM",
+          "email": "E-MAIL",
+          "status": "STATUT",
+          "archive": "Archive",
+          "unarchive": "Non archisif",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:227]: 😵‍💫 
```suggestion
          "unarchive": "Désarchiver",
```

File: frontend/assets/translations/fr.json [hunk 5]
```diff
@@ -1,921 +1,918 @@
...
+          "manageSso": "Gérer SSO",
+          "manageEnv": "Gérer les variables d'environnement"
+        },
+        "manageUsers": {
+          "usersAndPermission": "Utilisateurs et autorisations",
+          "inviteNewUser": "Inviter un nouvel utilisateur",
+          "name": "NOM",
+          "email": "E-MAIL",
+          "status": "STATUT",
+          "archive": "Archive",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:226]:
```suggestion
          "archive": "Archiver",
```

File: frontend/assets/translations/fr.json [hunk 6]
```diff
@@ -1,921 +1,918 @@
...
+            "udpateGroup": "Mettre à jour le groupe",
+            "addNewGroup": "Ajouter un nouveau groupe",
+            "enterName": "Entrez le nom",
+            "createGroup": "Créer un groupe",
+            "name": "Nom"
+          },
+          "permissionResources": {
+            "userGroup": "Groupe d'utilisateurs",
+            "apps": "applications",
+            "users": "Utilisatrice",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:248]:
```suggestion
            "users": "Utilisateurs",
```

File: frontend/assets/translations/fr.json [hunk 7]
```diff
@@ -1,921 +1,918 @@
...
+            "email": "e-mail",
+            "resource": "Ressource",
+            "createUpdateDelete": "Créer / mettre à jour / supprimer",
+            "folder": "Dossier"
+          }
+        },
+        "manageSSO": {
+          "manageSso": "Gérer SSO",
+          "generalSettings": {
+            "title": "réglages généraux",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:262]:
```suggestion
            "title": "Paramètres généraux",
```

File: frontend/assets/translations/fr.json [hunk 8]
```diff
@@ -1,921 +1,918 @@
...
+            "supportMultiDomains": "Prise en charge plusieurs domaines. Entrez les noms de domaine séparés par virgule. Exemple: Tooljet.com, ToolJet.io, YourOrganization.com",
+            "loginUrl": "URL de connexion",
+            "workspaceLogin": "Utilisez cette URL pour vous connecter directement à cet espace de travail",
+            "allowDefaultSso": "Autoriser SSO par défaut",
+            "ssoAuth": "Permettez aux utilisateurs de s'authentifier via SSO par défaut. Les configurations SSO par défaut peuvent être remplacées par le niveau d'espace de travail SSO."
+          },
+          "google": {
+            "title": "Google",
+            "enabled": "Activé",
+            "disabled": "Handicapée",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:276]: 😵‍💫 
link #4169

File: frontend/assets/translations/fr.json [hunk 9]
```diff
@@ -1,921 +1,918 @@
...
-            "maxHeightOfCanvas" :"Max height of canvas",
-            "backgroundColorOfCanvas":"Background color of canvas"
-
-        },
-        "Back":{
-            "text":"Back",
-            "tip" : "Back to Home"
-        }
+    "blankPage": {
+      "welcomeToToolJet": "Bienvenue à ToolJet!",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:356]:
```suggestion
      "welcomeToToolJet": "Bienvenue sur ToolJet !",
```

File: frontend/assets/translations/fr.json [hunk 10]
```diff
@@ -1,921 +1,918 @@
...
+    "and": "et"
+  },
+  "onBoarding": {
+    "finishToolJetInstallation": "Installation de jet d'outils de finition",
+    "organization": "Organisme",
+    "name": "Nom",
+    "email": "E-mail",
+    "receiveUpdatesFromToolJet": "Vous recevrez des mises à jour de l'équipe ToolJet (1-2 e-mails chaque mois, nous ne spammons pas)",
+    "finishSetup": "Configuration de la fin",
+    "skip": "Sauter"
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:423]:
```suggestion
    "skip": "Passer"
```

File: frontend/assets/translations/fr.json [hunk 11]
```diff
@@ -1,921 +1,918 @@
...
+      "disable": "Désactiver",
+      "borderRadius": "Rayon de frontière",
+      "boxShadow": "Boîte ombre",
+      "tooltip": "Infraction",
+      "showOnDesktop": "Afficher sur le bureau",
+      "showOnMobile": "Show sur mobile",
+      "showLoadingState": "Afficher l'état de chargement",
+      "backgroundColor": "Couleur de l'arrière plan",
+      "textColor": "Couleur de texte",
+      "loaderColor": "Couleur de chargeur",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:482]:
```suggestion
      "loaderColor": "Couleur du chargement",
```

File: frontend/assets/translations/fr.json [hunk 12]
```diff
@@ -1,921 +1,918 @@
...
+      "validation": "Validation",
+      "documentation": "{{componentMeta}} documentation",
+      "widgetNameEmptyError": "Le nom du widget ne peut pas être vide",
+      "componentNameExistsError": "Le nom du composant existe déjà",
+      "invalidWidgetName": "Nom du widget non valide. Devrait être unique et n'inclure que des lettres, des chiffres et un soulignement."
+    },
+    "commonProperties": {
+      "visibility": "Visibilité",
+      "disable": "Désactiver",
+      "borderRadius": "Rayon de frontière",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:474]: 😵‍💫 
```suggestion
      "borderRadius": "Rayon de la bordure",
```

**Review by roiLeo — COMMENTED** [review]
Hey! added some notes, there's still work to be done

File: frontend/assets/translations/fr.json (hunk 12 above)
Comment (akk312000) [reply frontend/assets/translations/fr.json]: @roiLeo , Can you please review the changes that I have made as you instructed, Thanks :)

File: frontend/assets/translations/fr.json [hunk 13]
```diff
@@ -1,921 +1,918 @@
...
+    "youHaveDisabled": "Vous avez handicapé",
+    "configureRedirectUrl": "Veuillez configurer l'URL de redirection du côté fournisseur SSO.",
+    "google": "Google",
+    "redirectUrl": "URL de redirection:",
+    "gitHub": "Github"
+  },
+  "oAuth2": {
+    "pleaseWait": "S'il vous plaît, attendez...",
+    "authSuccess": "Auth réussi, vous pouvez fermer cet onglet maintenant.",
+    "authFailed": "L'auteur a échoué"
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:446]:
```suggestion
    "authSuccess": "L'authentification à réussi, vous pouvez maintenant fermer cet onglet.",
    "authFailed": "L'authentification a échoué"
```

File: frontend/assets/translations/fr.json [hunk 14]
```diff
@@ -1,921 +1,918 @@
...
+    "multiWorkspace": "Espace multi-travaux",
+    "singleSignOnConfig": "Les configurations liées à la connexion unique sont déplacées des variables d'environnement vers la base de données. Veuillez référer ceci",
+    "link": "Lien",
+    "toConfigureSSO": "Pour configurer SSO.",
+    "haveGoogleGithubSSo": "Si vous avez des configurations SSO Google ou GitHub avant la mise à niveau et l'espace multi-works désactivé, les configurations de Thesso seront migrées pendant la mise à niveau, mais vous devez reconfigurer l'URL de redirection du côté fournisseur SSO. Les URL de redirection pour chaque SSO sont données ci-dessous.",
+    "isMultiWorkspaceEnabled": "Si vous avez activé Multi-Workspace, les configurations SSO ne seront pas migrées lors de la mise à niveau, vous devez donc reconfigurer le SSO sous l'espace de travail respectif.",
+    "youHaveEnabled": "Vous avez activé",
+    "setupSsoWorkspace": "Veuillez vous connecter avec le mot de passe et vous pouvez configurer SSO en utilisant Workspace",
+    "manageSsoMenu": "Gérer le menu SSO.",
+    "youHaveDisabled": "Vous avez handicapé",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:437]:
```suggestion
    "youHaveDisabled": "Vous avez désactivé",
```

File: frontend/assets/translations/fr.json [hunk 15]
```diff
@@ -1,921 +1,918 @@
...
+  },
+  "widgetManager": {
+    "commonlyUsed": "fréquemment utilisé",
+    "layouts": "disposition",
+    "forms": "formes",
+    "intregrations": "intégrations",
+    "others": "les autres",
+    "noResults": "Aucun résultat trouvé",
+    "tryAdjustingFilterMessage": "Essayez d'ajuster votre recherche ou votre filtre pour trouver ce que vous recherchez.",
+    "clearQuery": "requête claire"
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:456]:
```suggestion
    "clearQuery": "effacer la requête"
```

File: frontend/assets/translations/fr.json [hunk 16]
```diff
@@ -1,921 +1,918 @@
...
+  },
+  "oAuth2": {
+    "pleaseWait": "S'il vous plaît, attendez...",
+    "authSuccess": "Auth réussi, vous pouvez fermer cet onglet maintenant.",
+    "authFailed": "L'auteur a échoué"
+  },
+  "widgetManager": {
+    "commonlyUsed": "fréquemment utilisé",
+    "layouts": "disposition",
+    "forms": "formes",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:451]:
```suggestion
    "forms": "formulaires",
```

File: frontend/assets/translations/fr.json [hunk 17]
```diff
@@ -1,921 +1,918 @@
...
+    "intregrations": "intégrations",
+    "others": "les autres",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:453]:
```suggestion
    "others": "autres",
```

File: frontend/assets/translations/fr.json [hunk 18]
```diff
@@ -1,921 +1,918 @@
...
+      "invalidWidgetName": "Nom du widget non valide. Devrait être unique et n'inclure que des lettres, des chiffres et un soulignement."
+    },
+    "commonProperties": {
+      "visibility": "Visibilité",
+      "disable": "Désactiver",
+      "borderRadius": "Rayon de la bordure",
+      "boxShadow": "Boîte ombre",
+      "tooltip": "Infraction",
+      "showOnDesktop": "Afficher sur le bureau",
+      "showOnMobile": "Show sur mobile",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:478]:
```suggestion
      "showOnMobile": "Afficher sur mobile",
```

File: frontend/assets/translations/fr.json [hunk 19]
```diff
@@ -1,921 +1,918 @@
...
+      "documentation": "{{componentMeta}} documentation",
+      "widgetNameEmptyError": "Le nom du widget ne peut pas être vide",
+      "componentNameExistsError": "Le nom du composant existe déjà",
+      "invalidWidgetName": "Nom du widget non valide. Devrait être unique et n'inclure que des lettres, des chiffres et un soulignement."
+    },
+    "commonProperties": {
+      "visibility": "Visibilité",
+      "disable": "Désactiver",
+      "borderRadius": "Rayon de la bordure",
+      "boxShadow": "Boîte ombre",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:475]:
```suggestion
      "boxShadow": "Boîte d'ombre",
```

File: frontend/assets/translations/fr.json [hunk 20]
```diff
@@ -1,921 +1,918 @@
...
+      "markerColor": "Couleur de marqueur",
+      "showAxes": "Montrer les haches",
+      "showGridLines": "Montrer les lignes de grille",
+      "chartType": "Type de graphique",
+      "jsonDescription": "Description JSON",
+      "usePlotlyJsonSchema": "Utilisez Plotly JSON Schema",
+      "padding": "Rembourrage",
+      "hideTitleBar": "Masquer la barre de titre",
+      "hideCloseButton": "Masquer le bouton Fermer",
+      "hideOnEscape": "Se cacher",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:520]:
```suggestion
      "hideOnEscape": "Masquer sur Echap",
```

File: frontend/assets/translations/fr.json [hunk 21]
```diff
@@ -1,921 +1,918 @@
...
+      "modalSize": "Taille modale",
+      "open": "Ouverte",
+      "close": "proche",
+      "regex": "Expirat",
+      "minLength": "Longueur minimale",
+      "maxLength": "longueur maximale",
+      "customValidation": "Validation personnalisée",
+      "clear": "Dégager",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:528]:
```suggestion
      "clear": "Effacer",
```

File: frontend/assets/translations/fr.json [hunk 22]
```diff
@@ -1,921 +1,918 @@
...
+      "chartType": "Type de graphique",
+      "jsonDescription": "Description JSON",
+      "usePlotlyJsonSchema": "Utilisez Plotly JSON Schema",
+      "padding": "Rembourrage",
+      "hideTitleBar": "Masquer la barre de titre",
+      "hideCloseButton": "Masquer le bouton Fermer",
+      "hideOnEscape": "Se cacher",
+      "modalSize": "Taille modale",
+      "open": "Ouverte",
+      "close": "proche",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:523]:
```suggestion
      "open": "Ouvert",
      "close": "Fermer",
```

File: frontend/assets/translations/fr.json [hunk 23]
```diff
@@ -1,921 +1,918 @@
...
+      "secondaryTextColour": "Couleur de texte secondaire",
+      "min": "Min",
+      "max": "Max",
+      "value": "Évaluer",
+      "twoHandles": "Deux poignées",
+      "lineColor": "Couleur de la ligne",
+      "handleColor": "Manipuler la couleur",
+      "trackColor": "Couleurs de piste",
+      "timelineData": "Données de chronologie",
+      "hideDate": "Cacher",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:639]:
```suggestion
      "hideDate": "Cacher la date",
```

File: frontend/assets/translations/fr.json [hunk 24]
```diff
@@ -1,921 +1,918 @@
...
+      "description": "Afficher les données tabulaires paginées",
+      "columnType": "Type de colonne",
+      "columnName": "Nom de colonne",
+      "overflow": "Débordement",
+      "key": "clé",
+      "textColor": "Couleur de texte",
+      "validation": "Validation",
+      "regex": "Expirat",
+      "minLength": "Longueur minimale",
+      "maxLength": "longueur maximale",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:678]:
```suggestion
      "maxLength": "Longueur maximale",
```

File: frontend/assets/translations/fr.json [hunk 25]
```diff
@@ -1,921 +1,918 @@
...
+    "Debugger": {
+      "text": "Débogueur",
+      "tip": "Débogueur",
+      "errors": "les erreurs",
+      "noErrors": "Aucune erreur trouvée",
+      "clear": "dégager"
+    },
+    "Comments": {
+      "text": "commentaires",
+      "tip": "faire des commentaires",
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:899]:
```suggestion
      "tip": "afficher les commentaires",
```

File: frontend/assets/translations/fr.json [hunk 26]
```diff
@@ -1,921 +1,918 @@
...
+    },
+    "TreeSelect": {
+      "displayName": "Sélection de l'arborescence",
+      "description": "Sélectionnez des valeurs dans une vue d'arbre"
+    }
+  },
+  "leftSidebar": {
+    "Inspector": {
+      "text": "Inspectrice",
+      "tip": "Inspectrice"
```
Comment (roiLeo) [inline frontend/assets/translations/fr.json:882]:
```suggestion
      "text": "Inspecteur",
      "tip": "Inspecteur"
```

Comment (akk312000) [reply frontend/assets/translations/fr.json]: @roiLeo, did the changes are you instructed, please review for any mistakes.

**Comment by withshubh** [comment]
Hey @roiLeo 
Thanks for taking out time to review this PR. We really appreciate it. 🙌 
I would request you join our slack community and DM me: tooljet.com/slack

**Comment by withshubh** [comment]
Hey @akk312000 - Please resolve the merge conflicts.

**Comment by akk312000** [comment]
Done @withshubh .

**Review by withshubh — DISMISSED** [review]

**Comment by Navaneeth-pk** [comment]
@akk312000 please resolve the merge conflicts.

**Comment by akk312000** [comment]
@Navaneeth-pk, @withshubh   please check if there are any merge conflicts still present.

**Review by withshubh — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @akk312000 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**